exporter/datadogexporter/internal/logs/translator_test.go

```diff
@@ -0,0 +1,355 @@
+					l := plog.NewLogRecord()
+					l.Attributes().PutString("app", "test")
+					l.SetSeverityNumber(5)
+					return l
+				}(),
+				res: pcommon.NewResource(),
+			},
+			want: datadogV2.HTTPLogItem{
+				Message: *datadog.PtrString(""),
+				AdditionalProperties: map[string]string{
+					"app":              "test",
+					"status":           "debug",
+					otelSeverityNumber: "5",
+				},
+			},
+		},
+		{
+			name: "log_and_resource_with_attribute",
+			args: args{
+				lr: func() plog.LogRecord {
+					l := plog.NewLogRecord()
+					l.Attributes().PutString("app", "test")
+					l.SetSeverityNumber(5)
+					return l
+				}(),
+				res: func() pcommon.Resource {
+					r := pcommon.NewResource()
+					r.Attributes().PutString(conventions.AttributeServiceName, "otlp_col")
+					return r
+				}(),
+			},
+			want: datadogV2.HTTPLogItem{
+				Ddtags:  datadog.PtrString("service:otlp_col"),
+				Message: *datadog.PtrString(""),
+				Service: datadog.PtrString("otlp_col"),
+				AdditionalProperties: map[string]string{
+					"app":              "test",
+					"status":           "debug",
+					otelSeverityNumber: "5",
+				},
+			},
+		},
+		{
+			name: "log_with_service_attribute",
+			args: args{
+				lr: func() plog.LogRecord {
+					l := plog.NewLogRecord()
+					l.Attributes().PutString("app", "test")
+					l.Attributes().PutString(conventions.AttributeServiceName, "otlp_col")
+					l.SetSeverityNumber(5)
+					return l
+				}(),
+				res: func() pcommon.Resource {
+					r := pcommon.NewResource()
+					return r
+				}(),
+			},
+			want: datadogV2.HTTPLogItem{
+				Message: *datadog.PtrString(""),
+				Service: datadog.PtrString("otlp_col"),
+				AdditionalProperties: map[string]string{
+					"app":              "test",
+					"status":           "debug",
+					otelSeverityNumber: "5",
+					"service.name":     "otlp_col",
+				},
+			},
+		},
+		{
+			name: "log_with_trace",
+			args: args{
+				lr: func() plog.LogRecord {
+					l := plog.NewLogRecord()
+					l.Attributes().PutString("app", "test")
+					l.SetSpanID(spanID)
+					l.SetTraceID(traceID)
+					l.Attributes().PutString(conventions.AttributeServiceName, "otlp_col")
+					l.SetSeverityNumber(5)
+					return l
+				}(),
+				res: func() pcommon.Resource {
+					r := pcommon.NewResource()
+					return r
+				}(),
+			},
+			want: datadogV2.HTTPLogItem{
+				Message: *datadog.PtrString(""),
+				Service: datadog.PtrString("otlp_col"),
+				AdditionalProperties: map[string]string{
+					"app":              "test",
+					"status":           "debug",
+					otelSeverityNumber: "5",
+					otelSpanID:         fmt.Sprintf("%x", string(spanID[:])),
+					otelTraceID:        fmt.Sprintf("%x", string(traceID[:])),
+					ddSpanID:           fmt.Sprintf("%d", ddSp),
+					ddTraceID:          fmt.Sprintf("%d", ddTr),
+					"service.name":     "otlp_col",
+				},
+			},
+		},
+		{
+			// here SeverityText should take precedence for log status
+			name: "log_with_severity_text_and_severity_number",
```

Comment:
```suggestion
			name: "severity",
```
I'd see the test names more as an identifier rather than a description of the test.